dapp_runner/runner/runner.py

```diff
@@ -248,8 +248,32 @@ async def _listen_state_queue(self, service: DappService):
             except asyncio.CancelledError:
                 return
 
+            nodes_states = self.dapp_state
+
             # on a state change, we're publishing the state of the whole dapp
-            self.state_queue.put_nowait(self.dapp_state)
+            self.state_queue.put_nowait({
+                'nodes': nodes_states,
+                'app': self._get_app_state_from_nodes(nodes_states),
+                'timestamp': datetime.now(timezone.utc).isoformat()[:-6] + 'Z'
```

Comment:
maybe extract this call to `.._util`? so that we can easily reuse it and maintain consistent format in case we wish to include timestamps in other places? (e.g. the `data` stream)